bin/content-style

```diff
@@ -43,10 +55,10 @@ puts config.fetch('addendum', '') unless violations.empty?
 if csv
   csv_path = "#{command}/content-style-output.csv"
   output_csv = File.open(csv_path, 'w')
-  output_csv.puts ['File,Line,Violation']
+  output_csv.puts ['Test File,Test File Line,Test File Text,Violation,ERB File']
```

Comment:
Highly recommend the use of the [`CSV`](https://ruby-doc.org/stdlib-2.0.0/libdoc/csv/rdoc/CSV.html) module, to take care of certain subtleties of writing CSV (such as quoting, escaping delimiters, etc).